integration/src/integration/java/pl/allegro/tech/hermes/integration/management/SubscriptionManagementTest.java

```diff
@@ -126,6 +128,31 @@ public void shouldGetEventStatus() throws InterruptedException {
         traces.forEach(trace -> assertThat(trace).containsEntry("cluster", Configs.KAFKA_CLUSTER_NAME.getDefaultValue()));
     }
 
+    @Test
+    public void shouldReturnSubscriptionsThatAreCurrentlyTrackedForGivenTopic() {
+        // given
+        TopicName topic = new TopicName("tracked", "topic");
+        SubscriptionName subscription = createTrackedSubscription(topic);
+        operations.createSubscription(topic.getGroupName(), topic.getName(), "sub2", HTTP_ENDPOINT_URL);
+
+        // when
+        List<String> tracked = management.subscription().list(topic.qualifiedName(), true);
+
+        // then
+        assertThat(tracked).containsExactly(subscription.getName());
+    }
+
+    private SubscriptionName createTrackedSubscription(TopicName topic) {
+        SubscriptionName subscription = new SubscriptionName("sub", topic);
+        operations.createGroup(topic.getGroupName());
```

Comment:
there is single command for this -> `operations#build...(...)` creates group, topic and subscription in one run